dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -10182,8 +10216,10 @@ components:
             description: The status of the execution
             enum:
             - SUCCESSFUL
+            - FAILED
```

Comment:
What does `FAILED` mean, relative to the more specific `FAILED_` varieties?

Reply:
`FAILED` is a generic failure instead of a specific type of failure. Platforms may use this if they don't know or can't extract why it failed, like in Terra's case

Reply:
So, here's a brain dump regarding statuses and such, please feel free to ignore, it's a little late for me to be mentioning this sort of stuff... :)

I'm a little concerned about tailoring our statuses (or any of the other information) to one platform's way of doing things, and also, semantic overlap between the values.   Terra defines the generic failure and the "semantic" and "runtime" varieties, but almost certainly, some other platform is going to partition the status space differently.  A workflow run could fail because the data couldn't be loaded, or there weren't enough resources, etc...

So, how to deal with this?  One possibility is to keep the raw status data around, devise a status space we map all the data to, and perform the mapping as a postprocessing step, later.  Possibly, said postprocessing step knows about each metrics provider, and can handle the vagaries thereof?  Possibly, this happens in the aggregator...

If we wanted to encourage platforms to conform to our status definition, the data submission format could include both the basic status (SUCCESS, FAILURE, ABORT?) and a field for extended status information...